### DISCUSSION TITLE
Dropping forageables on Spring 1?

### DISCUSSION THREAD

**Discussion by waynn** [nathan-alden-sr/stardew-valley#8] [Q&A] [answered]
`Travel west. Just before you transition to The Farm, turn your character east, stand near the south side of the path, and drop all forageables on the ground. It may take several attempts to drop them as the game seems to choose random distances when throwing items.`

I didn't quite get the rationale for this. If you're already walking back to the farm, why not just drop everything off in the chest while you're going to clear a path to the Backwoods?

**Comment by nathan-alden-sr** [comment, marked as answer]
This is a great point. I have it there because that's what Zamiell's guide recommended; however, you are right that it seems useless.

https://github.com/nathan-alden-sr/stardew-valley/commit/6fc3cd60c185e1703f846aab8995651824ac45c5

**Reply by waynn** [reply]
Oh interesting, I didn't see it in Zamiell's guide, I thought you had added it. :+1:
